test.py

```diff
@@ -176,6 +176,14 @@ def test_pdf(self):
         self.assertEqual(self.gamma.pdf(4), (1 / (math.exp(2))),
                          'pdf function does not give expected result')
 
+    def test_cdf_lower(self):
+        self.assertEqual(round(self.gamma.cdf_lower(4), 5), .59399, 
+                         'cdf function does not give expected result')
+
+    def test_cdf_upper(self):
+        self.assertEqual(round(self.gamma.cdf_upper(4), 5), .40601, 
+                         'cdf function does not give expected result')
+
```

Comment:
Ah, @mhdzumair can you review this and check if the round() here adheres to our code style?

Reply:
See other distribution cdf function which is implemented with internal round arg.